licenses/README.md

```diff
@@ -0,0 +1,14 @@
+# Third Party Licenses
```

Comment:
Why don't we move this to  to `./THIRDPARTY.md`  like https://github.com/JuliaLang/julia/blob/master/THIRDPARTY.md and also directly link to the licenses like in the main Julia repo.

Reply:
That won't quite work here. It kind of works for Julia because when we acquire the third party source and binaries from BinaryBuilder during the build process, we also distribute the license files with it. In this case, we do not have that step to cover the terms of the Apache License.

Reply:
I see, thanks for the clarification.